README.md

```diff
@@ -1,10 +1,15 @@
-# Azure SDK for Node.js
+# Microsoft Azure SDK for Node.js - Cdn
 
-[![NPM version](https://badge.fury.io/js/azure.png)](http://badge.fury.io/js/azure) [![Build Status](https://travis-ci.org/Azure/azure-sdk-for-node.png?branch=master)](https://travis-ci.org/Azure/azure-sdk-for-node)
+This project provides a Node.js package for accessing the Azure Cdn Client. Right now it supports:
```

Comment:
This should be in the README.md file of your service`lib/services/cdnManagement/README.md`.It should not be at the top level README.md